uint32 id
uint32 seq
uint32 pickUp
uint32 delivery
float64 deadline